src/main/java/nl/novi/HwLes10ANWMTechItEasy/Controllers/TelevisionController.java

```diff
@@ -0,0 +1,75 @@
+package nl.novi.HwLes10ANWMTechItEasy.Controllers;
+
+import nl.novi.HwLes10ANWMTechItEasy.Exceptions.IndexOutOfBoundsException;
+import nl.novi.HwLes10ANWMTechItEasy.Exceptions.RecordNotFoundException;
+import nl.novi.HwLes10ANWMTechItEasy.Models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+
+@RestController
+public class TelevisionController {
+    private ArrayList<Television> televisionDataBase = new ArrayList<>();
+
+    public TelevisionController() {
+        //Prefill televisions collection met twee dummy's
+        Television t1 = new Television();
+        t1.setBrand("Samsung");
+        t1.setType("QD OLED 55S95B");
+
+        Television t2 = new Television();
+        t2.setBrand("Sony");
+        t2.setType("Bravia KD-43X85KP");
+
+        televisionDataBase.add(t1);
+        televisionDataBase.add(t2);
```

Comment:
Deze logic wil je niet in je constructor hebben.

  public List<Television> TELEVISION_ARRAYLIST = Arrays.asList(
            new Television("AB-1234", "Philips"),
            new Television("CD-5678", "LG")
    );